### PR TITLE
batch add members

### PR DESCRIPTION
**Pull request by JoshHare**

### CODE REVIEW

**Review by daniela-martinis — CHANGES_REQUESTED** [review]
merge conflicts noooooooooo

File: app/views/application/_nav.html.erb
Comment (daniela-martinis) [inline app/views/application/_nav.html.erb:1]: didn't you have an admin dashboard tab in here 🤔 otherwise looks good

**Review by daniela-martinis — APPROVED** [review]
ok goat looks good!

**Comment by JoshHare** [comment]
For Vishal <3

**Pull request merged**